Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.py

```diff
@@ -1221,48 +1444,340 @@ def search_records_command(args: dict[str, Any]) -> CommandResults:
+        owner=args.get("owner"),
+        criticality=str_to_dict(args.get("criticality")),  # type: ignore
+        assetDefinition=args.get("asset_definition"),
+        components=argToList(args.get("components")),
+        ttl=args.get("ttl"),
+    )
+    if args.get("context_output", "false") == "true":
+        return CommandResults(
+            readable_output=pretty_print_asset([result["data"]]),
+            outputs_prefix="Argus.Asset",
+            outputs=result,
+            raw_response=result,
+        )
+    return CommandResults(readable_output=pretty_print_asset([result["data"]]))
+
+
+def delete_asset_command(args: dict) -> CommandResults:
+    id_shortname = args.get("id_shortname")
+    if not id_shortname:
+        raise ValueError("id or shortname not given")
+
+    result = delete_asset(
+        idOrShortName=id_shortname,
+        customer=args.get("customer"),
+    )
+    if args.get("context_output", "false") == "true":
+        return CommandResults(
+            readable_output=pretty_print_asset([result["data"]]),
+            outputs_prefix="Argus.Asset",
+            outputs=result,
+            raw_response=result,
+        )
+    return CommandResults(readable_output=pretty_print_asset([result["data"]]))
+
+
+def update_asset_command(args: dict) -> CommandResults:
+    id_shortname = args.get("id_shortname")
+    if not id_shortname:
+        raise ValueError("id or shortname not given")
+
+    result = update_asset(
+        idOrShortName=id_shortname,
+        customer=args.get("customer"),
+        shortName=args.get("short_name"),
+        name=args.get("name"),
+        description=args.get("description"),
+        owner=args.get("owner"),
+        criticality=str_to_dict(args.get("criticality")),  # type: ignore
+        addComponents=argToList(args.get("add_components")),
+        deleteComponents=argToList(args.get("delete_components")),
+        ttl=args.get("ttl"),
+    )
+    if args.get("context_output", "false") == "true":
+        return CommandResults(
+            readable_output=pretty_print_asset([result["data"]]),
+            outputs_prefix="Argus.Asset",
+            outputs=result,
+            raw_response=result,
+        )
+    return CommandResults(readable_output=pretty_print_asset([result["data"]]))
+
+
+def get_vulnerability_by_asset_command(args: dict) -> CommandResults:
+    id_shortname = args.get("id_shortname")
+    context_output = args.get("context_output", "false")
+
+    if not id_shortname:
+        raise ValueError("Asset id or shortname not given")
+
+    result = get_vulnerability_by_asset(
+        asset=id_shortname,
+        customer=args.get("customer"),
+        includeDeleted=args.get("include_deleted"),
+        offset=args.get("offset"),
+        limit=args.get("limit"),
+    )
+    if context_output == "true":
+        return CommandResults(
+            readable_output=pretty_print_vulnerability([result["data"]]),
+            outputs_prefix="Argus.Asset",
```

Comment:
The context path prefix defined in the yml for that command are `Argus.Vulnerability`